fastqfil.py

```diff
@@ -0,0 +1,57 @@
+from additional_scripts.check_of_procedure import check_of_procedure
+from additional_scripts.type_of_chain import type_of_chain
+from additional_scripts.transcribe import transcribe
+from additional_scripts.reverse import reverse
+from additional_scripts.complement import complement
+from additional_scripts.reverse_complement import reverse_complement
+
+'''Accepts dictionaries with the sequence name,
+sequence and its quality of reading,
+filters according to the specified parameters
+of the percentage of GC nucleotides,
+the length of the sequences and the average quality 
+of reading. Returns a filtered dictionary'''
+def filter_fastq(seqs, gc_bounds = (0, 100), length_bounds = (0, 2**32), quality_threshold = 0):
+    filtered_fastq = dict()
+    if type(gc_bounds) == int:
+        gc_bounds = (0, gc_bounds)
+    if type(length_bounds) == int:
+        length_bounds = (0, length_bounds)
+    for name, sequal in seqs.items():
+        sequence, quality = sequal[0], sequal[1]

```

Comment:
```suggestion
    for seq_name, (sequence, quality) in seqs.items():

```

Скобки в данном случае очень важны, так как d.items() выдает нам 2 объекта - ключ и значение. Если сделать for seq_name, sequence, quality in d.items(), то python не сможет разбить 2 объекта по 3-м переменным.